quest/api/tools.py

```diff
@@ -94,10 +91,10 @@ def run_tool(name, options=None, as_dataframe=None, expand=None, as_open_dataset
     options = options or dict()
     options.update(kwargs)
 
-    plugin = load_plugins('tool', name)[name]
+    plugin = load_plugins(PluginType.TOOL, name)[name]
     result = plugin.run_tool(**options)
 
-    new_datasets = result.get('datasets', [])
+    new_datasets = result.get(UriType.DATASET, [])
```

Comment:
```suggestion
    new_datasets = result.get('datasets', [])
```
This is not referring to a URI type here, but is just accessing the key of a dictionary.